sdk/src/main/scala/io/horizen/account/websocket/WebSocketAccountConfigurator.scala

```diff
@@ -5,20 +5,20 @@ import jakarta.websocket.server.{HandshakeRequest, ServerEndpointConfig}
 import org.glassfish.tyrus.core.RequestContext
 import sparkz.util.SparkzLogging
 
+import java.net.InetAddress
 
 class WebSocketAccountConfigurator() extends ServerEndpointConfig.Configurator with SparkzLogging {
 
   override def modifyHandshake(config: ServerEndpointConfig, request: HandshakeRequest, response: HandshakeResponse): Unit = {
-    val remoteAddress = request.asInstanceOf[RequestContext].getRemoteAddr
-    if (!WebSocketAccountConfigurator.allowedIPs.contains(remoteAddress)) {
+    val remoteAddress = InetAddress.getByName(request.asInstanceOf[RequestContext].getRemoteAddr)
+    if (!remoteAddress.isAnyLocalAddress && !remoteAddress.isLoopbackAddress && !remoteAddress.isSiteLocalAddress && !WebSocketAccountConfigurator.allowedIPs.contains(remoteAddress.getHostAddress))
```

Comment:
It seems a bit different logic now.
I think we need to have `if allowedIps.isEmpty -> chech localhost` -> else check `WebSocketAccountConfigurator.allowedIPs.contains(remoteAddress.getHostAddress))`

Because now, we can't reject localhost connections, they are always accepted

Reply:
Ok sorry I missunderstood, I thought we wanted to allow localhost by default.